modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/play24/Play24CallFactory.mustache

```diff
@@ -163,20 +164,23 @@ public class Play24CallFactory implements okhttp3.Call.Factory {
 
                        @Override
                        public MediaType contentType() {
-                           return MediaType.parse(r.getHeader("Content-Type"));
+                           return Optional.ofNullable(r.getHeader("Content-Type"))
+                                          .map(MediaType::parse)
+                                          .orElse(null);
                        }
 
                        @Override
                        public long contentLength() {
-                           return r.getBody().getBytes().length;
+                           return r.asByteArray().length;
                        }
 
                        @Override
                        public BufferedSource source() {
-                           return new Buffer().write(r.getBody().getBytes());
+                           return new Buffer().write(r.asByteArray());
                        }
+                       
                    });
-
+                   
```

Comment:
FYI. I'll remove these spaces in my next commit.

Reply:
Ok